frontstage/templates/sign-in/sign-in.html

```diff
@@ -9,39 +9,33 @@
 
 {% block main %}
 
-    {% if errorType %}
-        {% set errorData = [] %}
+{% if errorType %}
+        {% set errorData = {} %}
+        {% set errorTitle = 'There is a problem with your answer' %}
         {% if errorType == 'failed' %}
```

Comment:
This error type "failed" is a bit strange and confusing, If I don't enter a password it won't come back as a fail, but by  definition it has failed